src/main/webapp/js/userMap.js

```diff
@@ -31,8 +31,11 @@ function handleData(err, countryCoordinates, userData) {
         }
     }
 
-    d3.select('#totalCount').text(userData.length + ' institutions from ' + countriesArr.length + ' countries');
-
+    //set the institution count in the page
+    document.getElementById('totalUserCount').innerHTML=userData.length;
+    //set the country count in the page
+    document.getElementById('totalCountryCount').innerHTML=countriesArr.length;
```

Comment:
There should be whitespace before and after the equal sign `=`. Likewise for the one before.